releng/org.eclipse.xtext.target/org.eclipse.xtext.target-r201809.target

```diff
@@ -0,0 +1,126 @@
+		<unit id="org.eclipse.xtext.testlanguages.source" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.testlanguages.ide" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.testlanguages.ide.source" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.tests" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.tests.source" version="0.0.0"/>
+		<repository location="http://services.typefox.io/open-source/jenkins/job/xtext-core/job/master/lastStableBuild/artifact/build/p2-repository/"/>
+	</location>
+	
+	<location includeAllPlatforms="false" includeMode="planner" includeSource="true" type="InstallableUnit">
+		<unit id="org.eclipse.xtext.builder.standalone" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.builder.standalone.source" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.common.types" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.common.types.source" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.ecore" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.ecore.source" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.generator" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.generator.source" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.java" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.java.source" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.purexbase" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.purexbase.source" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.purexbase.ide" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.purexbase.ide.source" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.smap" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.smap.source" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.xbase" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.xbase.source" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.xbase.ide" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.xbase.ide.source" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.xbase.testing" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.xbase.testing.source" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.xbase.testlanguages" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.xbase.testlanguages.source" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.xbase.testlanguages.ide" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.xbase.testlanguages.ide.source" version="0.0.0"/>
+		<repository location="http://services.typefox.io/open-source/jenkins/job/xtext-extras/job/master/lastStableBuild/artifact/build/p2-repository/"/>
+	</location>
+	
+	<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="false" type="InstallableUnit">
+		<unit id="org.eclipse.jdt.feature.group" version="0.0.0"/>
+		<unit id="org.eclipse.platform.feature.group" version="0.0.0"/>
+		<unit id="org.eclipse.pde.feature.group" version="0.0.0"/>
+		<unit id="org.eclipse.pde.api.tools.ee.feature.feature.group" version="0.0.0"/>
+		<repository location="http://download.eclipse.org/eclipse/updates/4.9-I-builds"/>
```

Comment:
this can not be changed yet since the release is not out yet

Reply:
https://github.com/eclipse/xtext/issues/1298

Reply:
Right. Also valid: http://download.eclipse.org/eclipse/updates/4.9milestones, but does not really make a difference